### ISSUE TITLE
Upgrade PyJWT dependency

### ISSUE THREAD

**Issue by juliocepar** [fireblocks/fireblocks-sdk-py#27]
Greetings. When I installed fireblocks-sdk with `pip install fireblocks-sdk`, I got the following error:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
flask-jwt-extended 4.2.1 requires PyJWT<3.0,>=2.0, but you have pyjwt 1.7.1 which is incompatible.
```
The error was solved when I downgrade flask-jwt-extended to version 3.25.1, but this is not what I want, since I need to use some features of flask-jwt-extended 4.x.x.

Please, upgrade fireblocks-sdk dependencies, or at least PyJWT in order to work with the latest version of each library. Thank you in advance.

It was tested in:
Python: 3.9.5
PIP: 21.1.1
OS: Windows 10 x64

**Comment by juliocepar** [comment]
Any feedback? @tomervil @michael-fireblocks @Yinon-Hever

**Comment by yarinvak** [comment]
included in v1.8.0 thanks for the suggestion
